pyvista/core/filters.py

```diff
@@ -4440,11 +4440,15 @@ def clean(poly_data, point_merging=True, tolerance=None, lines_to_points=True,
         else:
             return output
 
-    def geodesic(poly_data, start_vertex, end_vertex, inplace=False):
+    def geodesic(poly_data, start_vertex, end_vertex, inplace=False,
+                 keep_order=False):
```

Comment:
```suggestion
                 keep_order=True):
```

Let's just keep this default.